superset-frontend/src/explore/components/DataTablesPane/index.tsx

```diff
@@ -262,6 +282,9 @@ export const DataTablesPane = ({
         />
       );
     }
+    if (errorMessage) {
+      return <pre>{errorMessage}</pre>;
```

Comment:
@kgabryje Can you replace, in this PR or in a follow up, both instances of `<pre>` with the `Alert` component just like the chart container?